aws/table_aws_codepipeline_pipeline.go

```diff
@@ -0,0 +1,257 @@
+package aws
+
+import (
+	"context"
+
+	"github.com/aws/aws-sdk-go/aws"
+	"github.com/aws/aws-sdk-go/service/codepipeline"
+	"github.com/turbot/steampipe-plugin-sdk/grpc/proto"
+	"github.com/turbot/steampipe-plugin-sdk/plugin"
+	"github.com/turbot/steampipe-plugin-sdk/plugin/transform"
+)
+
+//// TABLE DEFINITION
+
+func tableAwsCodepipelinePipeline(_ context.Context) *plugin.Table {
+	return &plugin.Table{
+		Name:        "aws_codepipeline_pipeline",
+		Description: "AWS Codepipeline Pipeline",
+		Get: &plugin.GetConfig{
+			KeyColumns:        plugin.AllColumns([]string{"name", "region"}),
+			ShouldIgnoreError: isNotFoundError([]string{"PipelineNotFoundException"}),
+			Hydrate:           getCodepipelinePipeline,
+		},
+		List: &plugin.ListConfig{
+			Hydrate: listCodepipelinePipelines,
+		},
+		GetMatrixItem: BuildRegionList,
+		Columns: awsRegionalColumns([]*plugin.Column{
+			{
+				Name:        "name",
+				Description: "The name of the pipeline.",
+				Type:        proto.ColumnType_STRING,
+			},
+			{
+				Name:        "arn",
+				Description: "The Amazon Resource Name (ARN) of the pipeline.",
+				Hydrate:     getCodepipelinePipeline,
+				Type:        proto.ColumnType_STRING,
+				Transform:   transform.FromField("Metadata.PipelineArn"),
+			},
+			{
+				Name:        "created_at",
+				Description: "The date and time the pipeline was created, in timestamp format.",
+				Type:        proto.ColumnType_TIMESTAMP,
+				Transform:   transform.FromField("Created"),
+			},
+			{
+				Name:        "encryption_key",
+				Description: "The encryption key used to encrypt the data in the artifact store, such as an AWS Key Management Service (AWS KMS) key. If this is undefined, the default key for Amazon S3 is used.",
+				Hydrate:     getCodepipelinePipeline,
+				Type:        proto.ColumnType_JSON,
+				Transform:   transform.FromField("Pipeline.ArtifactStore.EncryptionKey"),
+			},
+			{
+				Name:        "artifact_stores",
+				Description: "A mapping of artifactStore objects and their corresponding AWS Regions. There must be an artifact store for the pipeline Region and for each cross-region action in the pipeline.",
+				Hydrate:     getCodepipelinePipeline,
+				Type:        proto.ColumnType_JSON,
+				Transform:   transform.FromField("Pipeline.ArtifactStores"),
+			},
```

Comment:
@ParthaI Move all `JSON` columns at the end

Reply:
@Subhajit97  I believe `encryption_key` column is one of the important column, should we move it to end?